frontend/src/__tests__/components/Categoties.test.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { Provider } from "react-redux";
+import { render } from "@testing-library/react";
+import configureMockStore from "redux-mock-store";
+import Categories from "../../components/Categories";
+
+const categories = [
+  {
+    id: 1,
+    name: "android",
+  },
+  {
+    id: 2,
+    name: "angular",
+  },
+  {
+    id: 3,
+    name: "aws",
+  },
+  {
+    id: 4,
+    name: "chrome",
+  },
+  {
+    id: 5,
+    name: "commandline",
+  },
+  {
+    id: 6,
+    name: "crystal",
+  },
+];
+
+describe("<Categories/>", () => {
+  it("renders properly", () => {
```

Comment:
regarding the DRY rule, mock store globally. 
Also use cleanup to unmounts the component from the container and destroys the container.

```suggestion
describe("<Categories/>", () => {
  const mockStore = configureMockStore([]);
  let store;
  
  beforeEach(() => {
    store = mockStore({
      categories,
    });
  });

  afterEach(cleanup);
  
  it("renders properly", () => {
```